lib/improver/threshold.py

```diff
@@ -171,16 +175,36 @@ def __init__(self, thresholds, fuzzy_factor=None,
             assert bounds[0] <= thr, bounds_msg
             assert bounds[1] >= thr, bounds_msg
 
-        self.below_thresh_ok = below_thresh_ok
+        # Lists of known logical comparisons. The final entry in each list
+        # must be recognisable by eval()
+        self.method_strings = {}
+        self.method_strings.update(dict.fromkeys(['ge', 'GE', '>='],
+                                                 {'function': operator.ge,
+                                                  'spp_string': 'above',
```

Comment:
As discussed, as we're not quite sure what we want this metadata to be, this could be deferred for the moment to a follow-on piece of work.